tests/test_create.py

```diff
@@ -565,13 +565,6 @@ def test_create_empty_env(self):
             assert stderr == ''
             self.assertIsInstance(stdout, str)
 
-    def test_anaconda_solve_time(self):
```

Comment:
Such performance/benchmarking tests are no bad idea, generally. Especially in light of all those index and solver changes for `4.6.x`.
It could make sense to add a `@pytest.mark.` to those and only run them on demand and/or in a separate, e.g., nightly, job. Or, if necessary, even put them into a separate test suite or repository.
I'd probably just use some `@pytest.mark.performance` and/or create a `test_performance.py` and set
```
import pytest
pytestmark = pytest.mark.performance
```
in there.